Cartridges/int_checkoutcom_sfra/cartridge/client/default/js/checkout/summary.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
```

Comment:
i don't think we should add this file to our integration. Please can you look into removing this file also.